src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  display: block;
+}
+
+.input-big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input-small {
+  box-sizing: border-box;
+  height: 42px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.big-search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small-search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.image-for-big-search {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.image-for-small-form {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.input:default {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
```

Comment:
you have a lot of similar styles 
use DRY principle (Don't Repeat Yourself)

Reply:
not fixed